app/src/main/java/com/asfoundation/wallet/ui/iab/LocalPaymentFragment.kt

```diff
@@ -60,7 +67,7 @@ class LocalPaymentFragment : DaggerFragment(), LocalPaymentView {
       throw IllegalArgumentException("domain data not found")
     }
   }
-  private val skudId: String? by lazy {
+  private val skudId: String by lazy {
```

Comment:
Can't the skuId be null?

Reply:
Yes it can, don’t know how it doesn’t crash when buying oil from trivial drive. Done